lib/api/core/plugins/pluginsManager.js

```diff
@@ -33,15 +33,35 @@ const
   CircularList = require('easy-circular-list'),
   fs = require('fs'),
   pm2 = require('pm2'),
+  clc = require('cli-color'),
+  semver = require('semver'),
   {
     KuzzleError,
     GatewayTimeoutError,
     PluginImplementationError,
     UnauthorizedError
   } = require('kuzzle-common-objects').errors;
 
+// This is default manifest for all plugin, can be overriden by "manifest.json" file at root path of each plugin
+const DefaultManifest = {
+  name: undefined,
+  version: '1.0.0',
```

Comment:
hum.. not sure about including a version in the manifest.
Do we actually need this information on Kuzzle side? 

What if the plugin also ships a package.json, in which the version is different? Which one should we pick?